README.md

```diff
@@ -1,6 +1,6 @@
 # ibm-block-csi-drive
 
-The Container Storage Interface (CSI) Driver for IBM block storage systems enables container orchestrators such as Kubernetes to manage the life-cycle of persistent storage.
+The CSI driver for IBM block storage enables container orchestrators, such as Kubernetes and Openshift, to manage the life-cycle of persistent storage.
```

Comment:
We forgot to update the readme per the latest name:  "IBM block storage CSI driver"

The IBM block storage CSI driver enables....